### ISSUE TITLE
Login Failed, Exiting

### ISSUE THREAD

**Issue by eightysixed** [Senso/eos-bot#1]
All of a sudden, I'm not able to login with the script (Ubuntu), or using the .EXE (Windows). I've deleted cookies.txt to no avail,  and even re-unzipped the Ubuntu package, as well as the Windows executable, but same thing. Did the developer possibly change the way users log in?

**Comment by Senso** [comment]
Yeah, I will have to fix the way the script logs in, as Ratan changed how login is done (POST instead of GET). He changed it because of a possible security issue I reported to him.

I fixed new_bot_eos.py, you will need to "git pull" the latest version. But the windows version will still be broken until I recompile it, in like 4 days.

So, was the bot working before that on Ubuntu?

**Comment by eightysixed** [comment]
Sounds good. I thought something probably changed server side. Yup, worked like a charm on Ubuntu. I never even downloaded the Windows Client until the it stopped working, just in case I messed something up on my end. 

I'm still not too competent with Linux, so unfamiliar with the 'git pull' command. All I did was download and extract the tarball, edit the config file, install the necessary libraries and modules on a fresh box and it worked perfectly. It doesn't look like you uploaded an updated Linux version yet, but I could be wrong. How can I get the updated script for use on the Linux box? (If not, I can wait until it's officially posted).

Thanks for the script! :)

edit - er, wait. Looks like new_bot_eos.py was updated. I'll pull that off tomorrow morning and report back. Thanks again! :)
